CONTRIBUTING.md

```diff
@@ -19,6 +19,8 @@ reported the issue. Please try to include as much information as you can. Detail
 * Any modifications you've made relevant to the bug
 * Anything unusual about your environment or deployment
 
+## Testing
+For running the integration tests locally, please add your AWS account number in the test_setup.py file.
```

Comment:
For a future PR, but is there another way we can have customers do this?  For a future PR, but ideally they could just use an environment variable (this should work out of the box unless you need to add account specific logic to this repo) 

We also should be adding some optional dependencies on anything needed to run the tests (I see boto3 at the very least, but there's likely more).  If none are needed, we should remove the unnecessary ones (IE replace boto3 with botocore).